cohort_4/playbook/ACTIVITIES.md

```diff
@@ -919,6 +919,10 @@ on Google (ex. if they don't know about the existence of the JavaScript
 console and it's the tool they need), then feel free to explain it to them
 directly. Use your discretion for when to do this.
 
+Moreover, we recommend that if you know that a hacker has solved the problem
+or think it is likely that another hacker may have the solution to this, you
+could recommend that the hacker reach out to one of their peers.
```

Comment:
Why? Should we add the reasoning behind this to this section?

Also, this wording feels pretty indirect.

> "We recommend...[that] you could recommend that the hacker reach out..."

What if we just said "We recommend...that you encourage hackers to reach out to their peers in this situation"?